operations/experimental/README.md

```diff
@@ -9,12 +9,113 @@ This is the README for Experimental Ops-files. To learn more about `cf-deploymen
 - For Community Ops-files, check out the [Community Ops-file README](../community/README.md).
```

Comment:
Rather than including the workflow in the experimental/README, I'd recommend 
1. creating a new file in the root of cf-deployment specifically for this workflow (something like "ops-file-promotion-workflow.md").
1. updating any existing links to point to the new workflow
1. reference and add a link to this workflow from both the operations and experimental README's
1. there's also a [Contributing to CF-Deployment](https://github.com/cloudfoundry/cf-deployment#contributing) section in the root README.md, it may be helpful to add a link from this section to this new workflow page as well.